apistar/hooks.py

```diff
@@ -65,6 +65,7 @@ def render_response(handler: Handler,
         renderer = negotiate_renderer(accept, renderers)
         if renderer is None:
             raise exceptions.NotAcceptable()
+            # return http.Response(status=406)
```

Comment:
Looks like this slipped in by mistake :)

Reply:
This commented line will make the test pass but it's certainly not the right way to fix the issue.